### PR TITLE
docs: add troubleshooting topic for containerd LimitNOFILE issue

### PR DESCRIPTION
**Pull request by nicofnt**
**Issue resolved by this Pull Request:**
Create new topic in the the troubleshooting guide to explain how to fix a new rook-ceph deployment that could remain in an unhealthy state or fails to config because of the LimitNOFILE value specified in the containerd.service configuration. See https://github.com/rook/rook/issues/14470 .

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

File: Documentation/Troubleshooting/ceph-common-issues.md [hunk 1]
```diff
@@ -774,3 +777,36 @@ data: {}
 ```
 
 If the ConfigMap exists, remove any keys that you wish to configure through the environment.
+
+## The cluster is in an unhealthy state or fails to configure
```
Comment (travisn) [inline Documentation/Troubleshooting/ceph-common-issues.md:781]: Is there any more specific title that would help describe this issue?

**Review by travisn — CHANGES_REQUESTED** [review]
The topic looks great, thanks for putting it together!

Please just squash the two commits, no need for a separate whitespace commit.

**Review by travisn — COMMENTED** [review]
Also it would be preferred to describe the issue in the [commit message](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure), instead of just refer to the issue number, thanks.

File: Documentation/Troubleshooting/ceph-common-issues.md (hunk 1 above)
Comment (nicofnt) [reply Documentation/Troubleshooting/ceph-common-issues.md:781]: Issue was pretty generic IMHO, hard to troubleshoot and the main symptoms are the ones described in the title. But I think I can be a little more specific.

**Review by travisn — APPROVED** [review]
Perfect, thanks for adding the new title.

**Pull request merged**